nextcloudappstore/core/views.py

```diff
@@ -270,3 +273,42 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['form'] = AppRegisterForm()
         return context
+
+
+class AppEditView(LoginRequiredMixin, TemplateView):
```

Comment:
Maybe so. I think I started out with one, but switched to a more general view because I was unhappy about something, I'll try to figure out what.